### ISSUE TITLE
Feature request: cleanup on an imported (non-live) CIB

### ISSUE THREAD

**Issue by jirib** [ClusterLabs/crmsh#1148]
``` shell
# rpm -q crmsh
crmsh-4.4.0+20221209.64abfaca-150400.3.12.1.noarch
```

After importing a customer CIB for analysis - `crm> cib import <file>` - I was thinking to remove past failures etc..., so `verify` does not show already "solved" (eg. via correcting cluster settings) errors.

But this is not possible - could this be considered?

``` shell
crm(cib.xml/jb154sapqe01)# resource cleaup
INFO: "cleaup" is accepted as "cleanup"
crm_resource: Error connecting to the controller: Connection refused
Error performing operation: Not connected
```

**Comment by liangxin1300** [comment]
Hi @jirib ,
When you got:
```
crm_resource: Error connecting to the controller: Connection refused
Error performing operation: Not connected
```
Is your cluster running?

And, if your cluster is running, I think after import, you need to run `crm cib commit`, 
Or the related resource couldn't be `real` running at the cluster nodes.
Then try to use `crm resource cleanup`

**Comment by jirib** [comment]
> Hi @jirib , When you got:
> 
> ```
> crm_resource: Error connecting to the controller: Connection refused
> Error performing operation: Not connected
> ```
> 
> Is your cluster running?
> 
> And, if your cluster is running, I think after import, you need to run `crm cib commit`, Or the related resource couldn't be `real` running at the cluster nodes. Then try to use `crm resource cleanup`

No, no cluster is running. I used `crm` in this context only for checking external CIBs.

**Comment by liangxin1300** [comment]
crm resource cleanup will call `crm_resource` command, which I think just works for the running cluster

**Comment by jirib** [comment]
That's why I used 'Feature request'; thus it would be nice to confider such functionality. As I said, I used `crm` for troubleshooting only (on this particular system).
